### ISSUE TITLE
[DOCS] - Cases API Curl Example

### ISSUE THREAD

**Issue by tammytorbert** [elastic/security-docs#1372]
## Description 

There are two issues when copying example as curl command from page (https://www.elastic.co/guide/en/security/8.0/cases-api-create.html)

- the description line ends up with a line break that causes an invalid json error.  The line break occurs between "email" and "banner" in the description field
- There is a comma after the value in the owner field.

## Acceptance Test Criteria

List all the ATC of each action and its intended result. 
For example: As a user, when [action (e.g., viewing, clicking, selecting, etc.)] I should [insert the expected result]. 

If the enhancement requires a procedure, include the steps.  

## Notes

- Add all appropriate labels to the issue, especially the version number label. 
- Be sure to add any necessary screenshots, code text or console commands for clarity. 
- Include any conditions or caveats that may affect customers.

**Comment by jmikell821** [comment]
Hi @tammytorbert, I'll see if shortening the description fixes the invalid json error. Also, are you saying that there _should not_ be a comma after the `owner` field? There are commas in all the examples listed, so I want to make sure that's what needs to be fixed. 
 
![2022-01-05_13-02-16](https://user-images.githubusercontent.com/57149392/148266735-84001bc2-1365-4bb4-b6da-8ffea7e86411.png)

**Comment by tammytorbert** [comment]
Yes, that's correct.  If i only fix the description, but leave the comma after the last value, I receive an "invalid json error".  I also validated the json with/without the comma after the value for owner with https://codebeautify.org/jsonviewer to help identify what was triggering the issue.  

Error sample: https://codebeautify.org/jsonviewer/cbe7c358
Fixed sample: https://codebeautify.org/jsonviewer/cb3e87cd
